artifactory/formats/buildpublishapi.go

```diff
@@ -0,0 +1,19 @@
+package formats
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+type BuildPublishOutput struct {
```

Comment:
I suggets merging the content of thsi file into the ```buildinfo/publish.go``` file, becuase it seems to be used by the build info publish command only.

Reply:
@eyalbe4 I created this file to mark that any change to this struct will break the API. Added a comment that clarifies it.
We discussed it before and agreed that this is the preferred way to protect APIs used externally (used by Azure in this case). Let me know what you think

Reply:
OK. NP.